spyder/plugins/projects/tests/test_plugin.py

```diff
@@ -289,5 +292,95 @@ def test_project_explorer_tree_root(projects, tmpdir, qtbot):
         assert topleft_index.data() == osp.basename(ppath)
 
 
+@pytest.mark.skipif(sys.platform == 'darwin', reason='Fails in macOS')
```

Comment:
I uploaded new packages for `watchdog` that include the missing library in macOS, so this shouldn't be necessary.

Please remove it to see what happens. If tests pass then this should be ready to go.